par: init at 1.5.2

jljox opened the pull request:
###### Motivation for this change

This an initial derivation. It's a dependency to working with Haskell using Vim.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X ] NixOS
  - [ ] OS X
  - [ X] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X ] Tested execution of all binary files (usually in `./result/bin/`)
- [ X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
## Opened https://github.com/NixOS/nixpkgs/issues/17521

jljox committed:
Paragraph reflow for email

Inspired by https://github.com/Homebrew/homebrew-core/blob/fb81645562f7e495a49561065912788af2d1f14f/Formula/par.rb

jljox committed:
par: init at 1.5.2

Inspired by https://github.com/Homebrew/homebrew-core/blob/fb81645562f7e495a49561065912788af2d1f14f/Formula/par.rb

jljox committed:
Merge branch 'master' into fix/par-initial-commit

ericsagnes commented:
If the patch is available online, `fetchpatch` should be preferred over copying the patch in nixpkgs source.

jljox committed:
Merge remote-tracking branch 'upstream/master'

jljox committed:
Merge branch 'master' into fix/par-initial-commit

jljox committed:
par: init at 1.5.2

Issue #17521
If the patch is available online, fetchpatch should be preferred over copying the patch in nixpkgs source.

jljox commented:
Using fetchpatch to retrieve the patch instead of a copy referenced from the git repo.

the-kenny commented:
Merged as 5262912
